src/node/index.ts

```diff
@@ -184,22 +189,20 @@ class Node {
         )
         return
       }
-      const file = this._getBinFile()
+      const file = await this._getBinFile()
       const cmd = global.platform.win32
         ? `set NODE_ENV=production&&"${file}"`
         : `NODE_ENV=production "${file}"`
-      this.logger.info(
-        `Launching point node md5: ${await md5File(file.toString())}`
-      )
-      return exec(cmd, (error, stdout, stderr) => {
-        if (stdout) this.logger.info('Ran successfully')
-        if (error) {
-          this.logger.error(ErrorsEnum.LAUNCH_ERROR, error)
-        }
-        if (stderr) {
-          this.logger.error(ErrorsEnum.LAUNCH_ERROR, stderr)
-        }
-      })
+      exec(cmd)
```

Comment:
why not?
```
try {
   await exec(cmd)
   doSomething()
} catch (e) {
  catchAction()
}
```

Avoid using .then and catch to have more uniform code

Reply:
Promise returned from `exec` will only resolve when the process is finished. So if I put `await`, `launch` function will never finish

Reply:
> Also I don't understand, why this check is skipped for Mac.
Because mac updates are now handled by `update-electron-app` (`src/main/index.ts`). There are no `if` statements to check for the different platforms because `update-electron-app` is "doing it internally" at the moment: there are no automatic updates for Linux, for Windows they are failing (no error, they just don't happen) and they are working for macOS.